README.md

```diff
@@ -18,15 +18,56 @@
-* I've included screenshots of what it is the student has to re-create in Interface Builder. Show them these screenshots and basically have them re-create it.
-* NO Autolayout (I've disabled it in the Xcode project that's included with this repo, so they don't have to worry about it).
+Now that you've learned how to get around Interface Builder, it's time to make your own app! Or, at least, the user interface for one. In this lab, you'll be laying out the graphical user interface for a Simon app. If you're not a child of the 80s and aren't familiar with Simon, here's the gist: Simon was an electronic game consisting of a disc with four buttons. Each button was a different color. The toy would randomly light up buttons in a random sequence, and then the player had to press the buttons in the same sequence. Sounds exciting, right? Not really, but there wasn't much else to do in the 80s except play Simon and watch He-Man! If you want the full run-down on the game, check out the [Simon](https://en.wikipedia.org/wiki/Simon_\(game\)) article on Wikipedia.
 
-![First SS](http://i.imgur.com/PdO6gU6.png)
+Simon is a pretty simple game, but at this point you may be thinking, "I can't write my own Simon game yet!" Well, maybe not _yet_, but you _will_ be able to soon. At any rate, for this lab, you're just going to be laying out the user interface—and that's something you know how to do!
 
-![Second SS](http://i.imgur.com/M2o9H9m.png)
+Here's what you're going to build in this lab:
 
-<a href='https://learn.co/lessons/InterfaceBuilderLab' data-visibility='hidden'>View this lesson on Learn.co</a>
+![Finished product](http://i.imgur.com/PdO6gU6.png)
 
+That might look a little bit daunting, but the interface is completely made up of view components you've already seen and used in previous lessons. Can you identify the views used in this graphical interface? Hint: There are 10 of them!
+
+Think for a minute...
+
+Really look at the interface...
+
+Did you find all 10? Here are the 10 views used in the interface:
+
+1. Grey background (a `UIView`)
+2. **Simon Says** label (a `UILabel`)
+3. A blue square (a `UIView`)
+4. An orange square (a `UIView`)
+5. A teal square (a `UIView`)
+6. A yellow square (a `UIView`)
+7. A **Blue** label (a `UILabel`)
+8. An **Orange** label (a `UILabel`)
+9. A **Teal** label** (a `UILabel`)
+10. A **Yellow** label (a `UILabel`)
+
+That's it! Not so complicated, is it? Let's walk through the creation of this app piece by piece. By the time you're done, you'll have something that looks like the above screenshot.
+
+1. Create a new Xcode project. (Hint: Use the **Single View Application** template from the iOS section.) You can call it **SimonSays**, or whatever you want.
+2. Select `Main.storyboard` in the file listing to show the main storyboard in Interface Builder's canvas.
```

Comment:
I'm somewhat torn on the issue of Auto Layout and Size Classes which are checked off (active) when you create a new project in Xcode. That being said, when they go to drag over the `UIViewController` onto the canvas it will appear a giant square as opposed to something that resembles an iPhone.

So, I think we can do one of two things.

1. Show them the necessary steps to uncheck the Use Auto Layout and Use Size Classes boxes located in the File Inspector when selecting your View Controller Scene from IB. In doing so (unchecking those boxes), the View Controller is no longer a square, it defaults to an iPhone 5S screen size. BUT, if you were to run it after laying out your views it still doesn't look the way it does in your canvas (because the simulator defaults to iPhone 6s plus). The student would then have to change the simulator to iPhone 5S, where the two will now match up and reflect exactly what the student is doing in IB.

2. Include an Xcode project where the above steps are completed for them, leaving them at the end of the day with an Xcode project with "No Scenes" in Interface Builder. When they open the Xcode project to complete the lab, when asked to drag a View Controller onto the Scene (Use Auto Layout and Use Size Classes) will by default already be checked off. As well, the simulator will already be selected to iPhone 5S for them where they can run it with no issues.